### DISCUSSION TITLE
SMILES import in standalone mode

### DISCUSSION THREAD

**Discussion by iwwwish** [epam/ketcher#566] [Q&A]
Dear Ketcher team,

First, thanks a lot for putting together Ketcher. I do use it on our prediction server. One of the feature I couldn't get to working is importing molecule as a SMILES string. It says "Extended SMILES and opening of Daylight SMILES is not supported in standalone mode." Is there a way around this?

I would like to be able to load a molecule into Ketcher from a text input (if not directly pasted from clipboard by doing Ctrl+v).

Thank you,
Vishal

<img width="772" alt="ketcher" src="https://user-images.githubusercontent.com/5487952/120061799-df288200-c02c-11eb-899f-a6be4cb9ccb4.png">

**Comment by AndreiMazol** [comment]
Dear @iwwwish ,

thank you for your interest in Ketcher.

Please provide an additional information:
- which version of Ketcher do you use? 
- which mode of Ketcher is used? (from Ketcher v2.2.x two different modes are available: remote - Indigo Service is required, and standalone - it is based on WASM. For details you  can look at https://www.npmjs.com/package/ketcher-standalone).

**Comment by iwwwish** [comment]
Dear @AndreiMazol,

Thanks for your reply. I am using Ketcher 2.0 in standalone mode. Is there a reason why SMILES import is disabled? I would appreciate if you could suggest a way around this problem.

Thank you,
Vishal

**Comment by AndreiMazol** [comment]
Dear @iwwwish ,

Ketcher 2.0 does not support standalone mode (now it is called offline remote mode, sorry for confusion). Now you have 2 options to work with SMILES:
1. Deploy Indigo Service and provide api_path query parameter to Ketcher instance:  http://localhost:3000?api_path={link_to_indigo_REST-Service_here} (SMILES, CML, InchL, SMARTS require working Indigo Service)
2. Switch to Ketcher 2.2.x (standalone mode). Artifacts can be found [here](https://github.com/epam/ketcher/releases/tag/v2.2.1)

Best regards,
Andrei
